src/types.ts

```diff
@@ -62,3 +71,33 @@ export type ChainID = number;
 export enum ColonyActionType {
   MintTokens = 'MINT_TOKENS',
 }
+
+export enum ColonyMotionType {
+  MintTokensMotion = 'MINT_TOKENS_MOTION',
+}
```

Comment:
I noticed in the corresponding CDapp PR you keep those merged, should we also stick to that idea here?